File: Cpp/From A to Z .cpp
// https://www.codewars.com/kata/6512b3775bf8500baea77663/train/cpp

#include <string>

using namespace std;

string gimme_the_letters(const string& sp)
{
    string result = "";

    for (int i = sp[0] ; i<=sp[2] ; i++)
    {
        result.push_back(char(i));
    }
    
    return result;
}

File: python/From A To Z .py
# https://www.codewars.com/kata/6512b3775bf8500baea77663/train/python


def gimme_the_letters(sp):
    result = ''
    for c in range(ord(sp[0]), ord(sp[2]) + 1):
        result += chr(c)
    return result
   

 
print(gimme_the_letters("a-z"))